output/linac_logger_drf_requests.txt

```diff
@@ -103,45 +68,13 @@ G:MVSTAT@p,30000,true
 G:PEAKS@p,30000,true
 G:PIDDB[2]@p,10000,true
 G:PIDDB[3]@p,10000,true
-G:PIDDB[8]@p,15000,true
```

Comment:
Any idea why these array elements were removed?

Reply:
Added a switch to output lists of devices dropped.  Result: `G:PIDDB[8]` and `G:PIDDB[9]` were suggested to be removed by K. Seiya. Maybe [2] and [3] were supposed to be ignored, too; I'll follow up with her.